### DISCUSSION TITLE
Can't compile with PlatformIO

### DISCUSSION THREAD

**Discussion by fpmirabile** [mriscoc/Ender3V2S1#575] [Q&A]
Hey everyone, trying to compile on my own the firmware, but I can't get it work.
I'm receiving this error on the terminal `Could not find one of 'package.json' manifest files in the package` 
Am I missing some library or something?

**Comment by mriscoc** [comment]
That is a problem with PlatformIO installation.

**Reply by fpmirabile** [reply]
I will check plataformio guides and come back with what I get

**Comment by mriscoc** [comment]
Duplicate of: https://github.com/mriscoc/Ender3V2S1/discussions/379
